modes:
  -
    control_axis:
      [
      # sliders
      0,  1,  2,  3,  4,  5,  6, 7, 8,
      # knobs
      17, 18, 19, 20, 21, 22, 23,
      ]

    control_buttons:
      [
      # up, down
      32, 33, 34, 35, 36, 37, 38, 39, 48, 49, 50, 51, 52, 53, 54, 55, 64, 65, 66, 67, 68, 69, 70, 71,
      # rew, play, ff, repeat, stop, rec
      43, 44, 42, 41, 45,
      # Track L, Track R, Marker Set, Marker Left, Marker Right, Cycle
      58, 59, 60, 61, 62, 46,
      ]
